docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -0,0 +1,187 @@
+---
+id: what-is-worker-versioning
+title: What is Worker Versioning?
+sidebar_label: Worker Versioning
+description: Worker Versioning allows you to more easily deploy changes to Workflow Definitions.
+tags:
+  - explanation
+  - versioning
+---
+
+Worker Versioning simplifies the process of deploying changes to [WorkflowDefinitions](/workflows/#workflow-definition). We recommend that you read about Workflow Definitions
+before proceeding, because Workflow Versioning is largely concerned with helping to manage
+nondeterministic changes to those definitions.
+
+This feature can help you manage nondeterministic changes by providing a convenient way to ensure
+that [Workers](/workers) with different Workflow & Activity Definitions operating on the same [TaskQueue](/tasks/#task-queue) do not attempt to process [Workflow Tasks](/tasks/#workflow-task) and
+[Activity Tasks](/tasks/#actvitiy-task) that they can't successfully process, according to sets of
+versions associated with that Task Queue that you define.
+
+You accomplish this goal by assigning a build identifier (a free-form string) to the code that
+defines a Worker, and specifying which build identifiers are compatible with each other by updating
+the version sets.
+
+### When and for what should you use Worker Versioning?
+
+The primary use case for this feature is the deployment of incompatible changes to short-lived
+[Workflows](/workers). On Task Queues using this feature, the Workflow starter does not need to know
+when new versions are introduced. New [Workflow Executions](/workflows#workflow-execution) run using
+the new code in the newly deployed Workers, and old (but still open) Workflow Executions are
+processed only by Workers with an appropriate version. Old workers can be decommissioned after the
+completion of any open workflows using their version.
```

Comment:
I think we shouldn't say "open" here since there's the issue of querying closed workflows.